client/my-sites/plugins/plugin-action/style.scss

```diff
@@ -52,7 +52,11 @@
 	cursor: pointer;
 
 	&.hide {
-		display: none;
+		visibility: hidden;
+
+		.plugin-action__label-text {
+			display: none;
+		}
 	}
 
```

Comment:
nit-pick: It seems that the `i` icon is not totally aligned by a couple of pixels 😅. 
<img width="155" alt="CleanShot 2022-09-19 at 19 15 15@2x" src="https://user-images.githubusercontent.com/3519124/191075036-fd8b9111-073b-4ee3-92a5-3b8c01c06205.png">


Adding a `display: flex` solves that but I will leave it to you to decide as it maybe looks better being top-aligned
```suggestion
.plugin-action__label-disabled-info {
	display: flex;
}
```

Reply:
Thanks for the suggestion, applied at https://github.com/Automattic/wp-calypso/pull/67984/commits/29a1de4023ee3ad7043830c071d9b9bec3a0b319